internal/state/test/factory/block.go

```diff
@@ -8,8 +8,8 @@ import (
 	"github.com/tendermint/tendermint/types"
 )
 
-func MakeBlocks(n int, state *sm.State, privVal types.PrivValidator) []*types.Block {
-	blocks := make([]*types.Block, 0)
+func MakeBlocks(n int, state *sm.State, privVal types.PrivValidator) ([]*types.Block, error) {
+	blocks := make([]*types.Block, 0, n)
```

Comment:
I'm not sure why we're appending here, since we know the exact capacity. I suggest:
```suggestion
	blocks := make([]*types.Block, n)
```